script/sync_prod_to_gcp/submissions_to_gcp.py

```diff
@@ -955,10 +955,22 @@ def submission_callback(message: Message) -> None:
             help_needed = os.environ.get("TEX_COMPILATION_RECIPIENT", "help@arxiv.org")
             subject = f"Uploading of {paper_id}v{version} failed"
             mail_body = f"Hello EUST,\nSubmission uploading for {paper_id}v{version} has failed. Please resolve the issue.\n\nThis message is generated by a bot on arxiv-sync.serverfarm.cornell.edu.\n"
-            cmd = ["/usr/bin/mail", "-r", "developers@arxiv.org", "-s", subject, help_needed]
+            SENDER = os.environ.get("SENDER", "developers@arxiv.org")
+            SMTP_SERVER = os.environ.get("SMTP_SERVER", "mail.arxiv.org")
+            SMTP_PORT = os.environ.get("SMTP_PORT", "25")
+            cmd = ["/usr/local/bin/arxiv-mail",
+                   "-t", help_needed,
+                   "-f", SENDER,
+                   "-m", SMTP_SERVER,
+                   "-p", SMTP_PORT,
+                   "-s", subject,
+                   ]
+            # using sendmail
+            # cmd = ["/usr/bin/mail", "-r", "developers@arxiv.org", "-s", subject, help_needed]
+
             mail = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
             try:
-                mail.communicate(mail_body, timeout=60)
+                mail.communicate(mail_body.encode('utf-8'), timeout=60)
```

Comment:
`mail_body` above is ascii - does the `.encode` call make any difference? Or is it future-proofing?

Reply:
The body text is no doubt 7-bit ascii. dosen't hurt to use utf-8. So "future proof" if you like.